packages/discovery-react-components/src/components/StructuredQuery/components/SatisfyRules/SatisfyRules.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from 'react';
+import { Dropdown } from 'carbon-components-react';
+import { Messages } from 'components/StructuredQuery/messages';
+import { structuredQuerySatisfyRulesDropdownClass } from 'components/StructuredQuery/cssClasses';
+import { formatSatisfyRulesDropdownMessage } from 'components/StructuredQuery/utils/formatSatisfyRulesDropdownMessage';
+
+export interface SatisfyRulesProps {
+  /**
+   * override default messages for the component by specifying custom and/or internationalized text strings
+   */
+  messages: Messages;
+}
+
+export const SatisfyRules: React.FunctionComponent<SatisfyRulesProps> = ({ messages }) => {
+  const handleOnChange = () => {
+    // TODO: Fully implement handling satisfy rules dropdown selections in a future issue
+  };
+
+  const satisfyRulesDropdownElement = (
+    <Dropdown
+      id="structured-query-dropdown"
+      items={[
+        messages.satisfyRulesDropdownAllOptionText,
+        messages.satisfyRulesDropdownAnyOptionText
+      ]}
+      type="inline"
+      initialSelectedItem={messages.satisfyRulesDropdownAllOptionText}
+      label={messages.satisfyRulesDropdownLabelText}
+      onChange={handleOnChange}
+      key="structured-query-dropdown"
```

Comment:
i don't think this `key` is needed

Reply:
I added this since we get a warning if we don't have the key here that we need it.